resources/js/components/Product/AddToCart.vue

```diff
@@ -304,14 +272,5 @@ export default {
             return disabledOptions
         },
     },
-
-    watch: {
-        customOptions: {
-            handler() {
-                this.calculatePrices()
-            },
-            deep: true,
-        },
-    },
```

Comment:
Todo: Will probably need to re-implement such a watcher to have the options update in the price component